### ISSUE TITLE
New mod request: streets

### ISSUE THREAD

**Issue by webD97** [minetest-mods/minetest-mods.github.io#117]
We (@Thomas--S and me) would like to move [streets](https://github.com/minetest-streets/streets) into this organization since we both have not had enough time fore the last years to properly maintain the mod. The licenses are MIT for code and CC-BY-SA 3.0 for media. Would it be possible to grant admin access to both of us afterwards?

**Comment by sofar** [comment]
Sorry, I seemed to have missed this entirely. This is entirely ok with me. If you are still wanting to migrate, you can initiate a transfer to `sofar`. Then I will move it through into this project. Access rights will remain the same during the process.

**Comment by wsor4035** [comment]
commonly accepted version in the community now is https://content.minetest.net/packages/cheapie/roads/

**Comment by sofar** [comment]
Any comment from @Thomas--S or @webD97 ? Is `roads` the way forward?

**Comment by webD97** [comment]
Hm, I haven't been active in the community for a long time now and I don't know if there is a "commonly accepted version in the community". From what I can tell from ContentDB (https://content.minetest.net/packages/webdesigner97/streets/, https://content.minetest.net/packages/cheapie/roads/), both mods had their last release in 2018 but streets has far more downloads (13k vs 4k).

I don't know of any way to find out the "truth" here. Roads looks well polished to me and I think it wouldn't make much sense to have both mods here. If cheapie is still actively developing roads, I think it could be the way forward. If not, we should go with streets and see if we can merge back roads' additions into streets.

**Comment by Thomas--S** [comment]
I agree with webD97 that roads is not necessarily the commonly accepted version of streets.
Thus, I still support the transfer request.

**Comment by webD97** [comment]
The transfer has started

**Comment by sofar** [comment]
completed.
